docs/config.md

```diff
@@ -15,7 +15,7 @@ rules:
     severity: ERROR
 ```
 
-This is a minimal rule. It will be run on any Python code, and will output the rule `id` "eqeq-is-bad" along with the `message` and the line and column range the match occurred on. So it will match code like `1 == 1` and output the message: "Dude, 1 == 1 is always true!"
+When invoked with `sgrep -f myconfig.yaml project/`, this rule will run on Python files inside `project/` and output the rule `id` "eqeq-is-bad" along with the `message` and the line and column range the match occurred on. Given a line inside a Python file with `1 == 1`, sgrep will output the message: "1 == 1 is always true!"
```

Comment:
```suggestion
When invoked with `sgrep -f myconfig.yaml project/`, this rule will run on Python files inside `project/` and outputs the rule `id` "eqeq-is-bad" along with the `message` and the line and column range the match occurred on. For example, given a line inside a Python file with `1 == 1`, sgrep will output the message: "1 == 1 is always true!"
```

Reply:
I think this change actually makes it unnecessarily wordier but maybe I'm missing something

Reply:
Agreed, no need to make it wordier.